### ISSUE TITLE
Global browserslist should be able to resolve local installed configs

### ISSUE THREAD

**Issue by lemnisSP** [browserslist/browserslist#241]
I have browserslist globaly installed and had an hard time to figure out why a browserslist-config-* project couldn't be resolved.

Within package.json
```
"browserslist": [
  "extends browserslist-config-vue"
],
```

When running `browserslist`
```
C:\Users\Lisa\AppData\Roaming\npm\node_modules\browserslist\cli.js:82
      throw e
      ^

Error: Cannot find module 'browserslist-config-vue'
    at Function.Module._resolveFilename (module.js:536:15)
    at Function.Module._load (module.js:466:25)
    at Module.require (module.js:579:17)
    at require (internal/module.js:11:18)
    at Object.loadQueries (C:\Users\Lisa\AppData\Roaming\npm\node_modules\browserslist\node.js:83:19)
    at Function.select (C:\Users\Lisa\AppData\Roaming\npm\node_modules\browserslist\index.js:632:26)
    at C:\Users\Lisa\AppData\Roaming\npm\node_modules\browserslist\index.js:151:33
    at Array.reduce (<anonymous>)
    at resolve (C:\Users\Lisa\AppData\Roaming\npm\node_modules\browserslist\index.js:132:18)
    at browserslist (C:\Users\Lisa\AppData\Roaming\npm\node_modules\browserslist\index.js:224:16)
```

When running `node ./node_modules\browserslist\cli.js`
```
and_chr 64
and_ff 57
and_qq 1.2
and_uc 11.8
android 62
android 4.4.3-4.4.4
android 4.4
android 4.2-4.3
baidu 7.12
...
```

**Comment by ai** [comment]
Hi. Thanks for the report.

Seems like we need to pass `path` to `loadQueries` and do `require.resolve` before `require`. https://github.com/browserslist/browserslist/blob/master/node.js#L103

Do you want to make PR and save your name in the history? Or I can do it in the Friday.

**Comment by lemnisSP** [comment]
If you could do it would be great

**Comment by ai** [comment]
Sure :)

**Comment by ai** [comment]
Fixed 648cc2f

**Comment by ai** [comment]
Released in 3.2.4
https://twitter.com/Browserslist/status/979780404201828354
